src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+  let result = 0;
+
+  if (days < 3) {
+    result = sum;
+  } else {
+    if (days >= 3 && days < 7) {
+      result = sum - 20;
+    } else {
+      result = sum - 50;
+    }
```

Comment:
Do you really need to store this values into result?
Try to return the full price immediately after check. It will help to avoid redundant else...if.
Also, create a variables for all main values (20, 50...) to make your code cleaner

Reply:
Ok, I got it. Thank you!